submodules/strategyHolidays/strategyHolidays.js

```diff
@@ -276,7 +338,26 @@ define(function(require) {
 
 			template.on('click', '.import-csv', function(event) {
 				event.preventDefault();
-				self.strategyHolidaysImportDatesFromCsvData(parent);
+				self.strategyHolidaysImportHolidaysFromCsvData(parent);
+			});
+
+			template.on('click', '.include-national-holidays', function(event) {
+				event.preventDefault();
+
+				var dateHolidays = new DateHolidays(),
+					allCountries = dateHolidays.getCountries(),
+					countryCode = _.last(_.split(monster.config.whitelabel.language, '-')),
```

Comment:
Since this operation is done twice, it might be a good idea to extract it in its own function, which would give the benefit of clarifying what we are trying to achieve:
```suggestion
	parseCountryCode = _.flow(
		_.partial(_.split, _, '-'),
		_.last
	),
	countryCode = parseCountryCode(monster.config.whitelabel.language),
```
What do you think?

Reply:
This is a good suggestion, thank you!